site/content/en/docs/technical-outcomes/scheduling/limit-range/_index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Scenario 4"
+---
+
+# Scenario 4: Scheduling pods with a limit range
+
+A limit range schedule policy can be used in a KWOK cluster.
+
+## Prerequisites
+
+- KWOK must be installed on the machine. See [installation](https://kwok.sigs.k8s.io/docs/user/installation/).
+- Install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/)
+
+## Create cluster
+
+```bash
+kwokctl create cluster
+```
+
+## View clusters
+
+This ensures that the cluster was created successfully.
+
+```bash
+kwokctl get clusters
+```
+
+## Deploy node
+
+- Node: 1 worker node in the cluster
+
+{{< expand "node.yaml" >}}
+
+{{< code-sample file="node.yaml" >}}
+
+{{< /expand >}}
+
+```bash
+kubectl apply -f node.yaml
```

Comment:
```suggestion
kwokctl scale node --replicas 1
```

Reply:
This means that I'll have to delete `node.yaml` and also remove its `expand` file reference, correct?